src/App.jsx

```diff
@@ -14,45 +17,125 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FIELD_ABC = 'abc';
+const SORT_FIELD_LENGTH = 'length';
+let REVERSE = false;
+let ABC = false;
+let LENGTH = false;
+
+export const App = () => {
+  let visibleGoods = [...goodsFromServer];
+  const [count, setCount] = useState(0);
+  const [sortField, setSortField] = useState('');
+  const hasResetButton = REVERSE === true || sortField !== '';
```

Comment:
```suggestion
  const hasResetButton = REVERSE || !!sortField;
```
pls see [falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)